ts/src/test/Exchange/base/test.market.ts

```diff
@@ -105,7 +105,33 @@ function testMarket (exchange, skippedProperties, method, market) {
         // linear & inverse should have different values (true/false)
         // todo: expand logic on other market types
         if (isSwapOrFuture) {
-            assert (market['linear'] !== market['inverse'], 'market linear and inverse must not be the same' + logText);
+            // todo:
+            if (!('quanto' in market)) {
```

Comment:
@ttodua I don't think we should this check because the market might have `quanto = false` if this field ends up in the market structure

Reply:
I mean that a regular (non quanto) market might have quanto:false, so the else would run because the `key` `quanto` exists

Reply:
yes, the `else` will run, be this already happens inside `if is swap/futures` clause, so spot will not be here

Reply:
@ttodua I'm not talking about spot but regular swap markets, for instance the market BTC/USDT:USDT might have `quanto = false/undefined` and in that case you're checking the `key` only not the value

Reply:
yes, but that's what the test should catch.
if `quanto` is false, then tests will pass. only if it's `undefined` the test will throw a failure. we shouldnt have undefined for a swap market - it's clearly either linear, inverse or quanto. none of them is undefined in any exchange

Reply:
ah, wait, let me make an edit...

Reply:
it is ok now?